.travis.yml

```diff
@@ -36,19 +47,32 @@ matrix:
         - USE_PROJECT=1
         - USE_VTK=0
         - USE_OMP=1
-        - ENABLE_PACKAGING=1
         - BLASLAPACK_IMPLEMENTATION=LAPACK
     - os: linux
       compiler: gcc
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
-        - USE_VTK=1
+        - USE_VTK=0
         - USE_OMP=1
         - USE_GIFTI=1
         - USE_COVERAGE=1
```

Comment:
I would not put coverage on for this build. I would activate it just for the most complete build. wdyt?

Reply:
doesn't it merge them ?